src/DGtal/shapes/parametric/AccFlower2D.h

```diff
@@ -61,7 +61,7 @@ namespace DGtal
    * NB: A backport from [ImaGene](https://gforge.liris.cnrs.fr/projects/imagene).
    */
   template <typename TSpace>
-  class AccFlower2D:  public StarShaped2D<TSpace>
+  class AccFlower2D final:  public StarShaped2D<TSpace>
```

Comment:
Yes. It forbids further derivation. The reason is that we do use polymorphism for Euclidean shapes and star-shaped, while at the same using genericity. Genericity does not call virtual destructor while polymorphism should do it. I think it was related to GaussDigitizer and the ConstAlias mecanism. And the problem was only detected in VC++ (it was not a problem in all our uses, but it could have been if we add use a GaussDigitizer to digitize a subclass of Ellipse2D for instance).
If you don't like these "final" (which allows to avoid virtual destructors), we can remove the ConstAlias mecanism. I let you decide.